SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -41,6 +41,7 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 
 	virtual TSet<VirtualWorkerId> GetVirtualWorkerIds() const PURE_VIRTUAL(UAbstractLBStrategy::GetVirtualWorkerIds, return {};)
 
+	virtual bool ShouldRunBeginPlayWithAuthority(const AActor& Actor) const { return false; }
```

Comment:
uber nit: not sure if I love this name - aren't we just asking if the Actor is in my LB zone? Do we need to bring BeginPlay into it? Or is the expectation that strategies will overload this specifically for the BeginPlay functionality?

Reply:
Changed to just use existing (renamed) functions